spec/merchant_spec.rb

```diff
@@ -0,0 +1,9 @@
+require './lib/merchant'
+
+RSpec.describe Merchant do
+  it 'exists' do
+    m = Merchant.new({:id => 5, :name => "Turing School"})
+
+    expect(m).to be_instance_of(Merchant)
+  end
+end
```

Comment:
It may seem redundant, but maybe create tests to ensure our attr_readers are working and returning the values we expect.

Reply:
Caleb brings up a good point. Checking to see if the attr_readers are working is probably a good idea here.